support/latest/severity.md

```diff
@@ -2,11 +2,23 @@
-|2 - Major Impact|A Severity Two issue indicates the production system is functioning but severely impacted with limited capabilities, or is unstable with major periodic interruptions, or mission critical applications, while not being affected, have experienced material system interruptions. A Severity Two issue may also indicate there is a time sensitive question impacting performance or deliverables, or a major subsystem under development is blocked. Alfresco assigns a specialist, and provides additional, escalated procedures as determined necessary by Alfresco Support Services staff. Alfresco exercises commercially reasonable efforts to provide a workaround or include a fix for the Severity Two issue in the next Service Pack/Maintenance Release.|
-|3 - Minor Impact|A Severity Three issue indicates the production system is still functioning but capabilities are moderately impacted, or the system is unstable with minor periodic interruptions or a minor loss of product functionality. A Severity Three issue can also indicate there are errors in system development that may be impacting performance deliverables. There is a medium-to-low impact on your business, but your business continues to function, including by using a procedural workaround.|
-|4 - Question - How To - Enhancement|A Severity Four issue is a general usage question, request for information, reporting of a documentation error, or recommendation for a future product enhancement or modification.  There is low-to-no impact on your business or the performance or functionality of your system.|
+The outlined Hyland Support severity definitions are utilized when prioritizing and working on your issue. Reserve communication of Severity Level One (1) for your most critical issues (production environment down) to ensure the Hyland Support team can respond to all issues appropriately.
+
+| Severity | Definition |
+|--------- | ---------- |
+| Level 1 | "Level 1" means any Error that causes total or substantial software failure, which means that the software is down and the customer is unable to access the software in any way within their production environment. |
+| Level 2 | "Level 2" means an Error that causes substantial software failure which prevents a portion of a customer’s users from accessing the software in any way within the production environment. |
```

Comment:
The numbering appears weird on mine. Level 1 is ok but Level 2 onwards the 2 goes onto the line underneath.

Reply:
I don' think we can fix this